Pages/BluetoothPage.xaml

```diff
@@ -8,19 +8,28 @@
     <ContentPage.BindingContext>
         <local:ViewModels.BluetoothViewModel/>
     </ContentPage.BindingContext>
-
+    
     <ContentPage.Content>
+
         <StackLayout BackgroundColor="#333" Orientation="Vertical">
 
             <Image Source="enlighten_logo.png" WidthRequest="300"/>
 
-            <!-- Button Bar -->
-            <Frame CornerRadius="10" BackgroundColor="#666" Padding="2">
-                <Frame CornerRadius="10" BackgroundColor="#444" Padding="2">
-                    <StackLayout Orientation="Horizontal" Padding="2" HorizontalOptions="FillAndExpand" x:DataType="local:ViewModels.BluetoothViewModel">
+            <StackLayout BackgroundColor="#333" HorizontalOptions="Center" Orientation="Horizontal">
+                <Label Text="Bluetooth" VerticalOptions="Center" TextColor="#cccccc"/>
+                <RadioButton IsChecked="{Binding usingBluetooth}" BorderColor="#cccccc"/>
+                <Label Text="USB" VerticalOptions="Center" TextColor="#cccccc"/>
+                <RadioButton IsChecked="{Binding usingUSB}" BorderColor="#cccccc"/>
```

Comment:
Interesting. So the user can choose whether the "BluetoothPage" (todo, "ConnectionPage") displays either USB or BLE devices. I like it.

Reply:
yes! I don't love how the control looks here but it does the job for now, and I as I said in the commit it isn't tested in USB yet but I'll test when I have the appropriate hardware (next week, I only have a usb micro android right now and no micro to c cable, obviously)